### ISSUE TITLE
Prevent overlay from flickering when desktop switches

### ISSUE THREAD

**Issue by cspotcode** [cspotcode/virtual-desktop-grid-switcher#12]
We can use the same trick formerly used for sticky windows.  I think we'll have to stop using SplashImage and re-implement the overlay with Gui commands.  Another benefit of re-implementation is that we can properly center the icon and tweak the colors and border and whatnot.

**Comment by seraasch** [comment]
Would be cool if the "grid" represented the shape in the configuration also... E.g. I use a 6x1 grid that shows up as a 3x3 in the overlay

**Comment by cspotcode** [comment]
True, but that's a different issue.  Can you submit it separately?
